ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/NetworkProtocolVersion.hs

```diff
@@ -0,0 +1,34 @@
+{-# LANGUAGE TypeFamilies #-}
+
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Ouroboros.Consensus.Shelley.Ledger.NetworkProtocolVersion (
+    ShelleyNodeToClientVersion(..)
+  ) where
+
+import           Data.List.NonEmpty (NonEmpty (..))
+
+import qualified Ouroboros.Network.NodeToClient as N
+
+import           Ouroboros.Consensus.Node.NetworkProtocolVersion
+
+import           Ouroboros.Consensus.Shelley.Ledger.Block
+
+data ShelleyNodeToClientVersion =
+    -- | No local state query protocol
+    ShelleyNodeToClientVersion1
+
+    -- | Use local state query protocol
+  | ShelleyNodeToClientVersion2
```

Comment:
No need for both, we don't need to support "No local state query protocol". So just keep `ShelleyNodeToClientVersion1`, remove the comment and the other constructor.